controlplane/kubeadm/internal/workload_cluster_etcd_test.go

```diff
@@ -508,16 +508,18 @@ kind: ClusterStatus`,
 
 	tests := []struct {
 		name                string
-		objs                []runtime.Object
+		objs                []client.Object
```

Comment:
This was also added automatically as part of the backporting, I hope this should be okay

Reply:
I don't think that is going to compile

Reply:
Yeah, the slice has to be []runtime.Object here